homeassistant/components/airvisual_pro/__init__.py

```diff
@@ -0,0 +1,106 @@
+"""The AirVisual Pro integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+from typing import Any
+
+from pyairvisual import NodeSamba
+from pyairvisual.node import NodeProError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_IP_ADDRESS, CONF_PASSWORD, Platform
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity import DeviceInfo, EntityDescription
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+    UpdateFailed,
+)
+
+from .const import DOMAIN, LOGGER
+
+PLATFORMS = [Platform.SENSOR]
+
+DEFAULT_UPDATE_INTERVAL = timedelta(minutes=1)
```

Comment:
Is this adjustable? Default implies that it can be changed, should it be `UPDATE_INTERVAL`?